app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -495,7 +495,11 @@ struct DeviceDelegateOpenXR::State {
 
     uint32_t displayRefreshRateCount = 0;
     CHECK_XRCMD(OpenXRExtensions::sXrEnumerateDisplayRefreshRatesFB(session, 0, &displayRefreshRateCount, nullptr));
-    CHECK(displayRefreshRateCount > 0);
+    if (displayRefreshRateCount <= 0) {
```

Comment:
Given that displayRefreshRateCount is uint32_t, why comparing against negative values ?

Reply:
Right, it's OK to have just == 0